config.json

```diff
@@ -37,6 +37,44 @@
         }
       }
     },
+    "cnpm": {
+      "default": "9.2.0+sha1.f4ced5247b703103c6fc7b6e7ec2927c8f0a3d21",
+      "fetchLatestFrom": {
+        "type": "url",
+        "url": "https://registry.npmmirror.com/cnpm",
+        "fields": {
+          "tags": "latest",
+          "versions": "tags"
+        }
+      },
+      "transparent": {
+        "commands": [
+          [
+            "cnpm",
+            "sync"
+          ]
+        ]
+      },
+      "ranges": {
+        "*": {
+          "url": "https://cdn.npmmirror.com/packages/cnpm/{}/cnpm-{}.tgz",
+          "bin": {
+            "cnpm": "./bin/cnpm"
+          },
+          "commands": {
+            "use": ["cnpm", "install"]
+          },
+          "registry": {
+            "type": "url",
+            "url": "https://registry.npmmirror.com/cnpm",
+            "fields": {
+              "versions": "versions"
+            }
+          },
+          "post-install": "npm i --registry=https://registry.npmmirror.com --production"
```

Comment:
We can't expect that `npm` will be available on the PATH

Reply:
And for perf and stability reasons I don't think we should run an install either.

All existing package managers are distributed with all their dependencies, one way or another.

Reply:
👌🏻 I will try to bundle the deps together in a .tgz file by enabling the bundleDeps configuration,
and also add dependencies for existing versions.